README.md

```diff
@@ -251,6 +251,11 @@ The following resources include cisco types and providers along with cisco provi
   * [`cisco_pim_grouplist`](#type-cisco_pim_grouplist)
   * [`cisco_pim_rp_address`](#type-cisco_pim_rp_address)
 
+* Multisite Types

```

Comment:
Would EVPN Multisite Types be more descriptive?  My worry is that multisite by itself may be to generic.